### DISCUSSION TITLE
how to add new site как добавить новый сайт

### DISCUSSION THREAD

**Discussion by kiiroless** [bol-van/zapret#948] [Q&A]
how to add new site so zapret would work on it too. it for animego and jut.su. they been blocked and only vpn could help with that. the question is how i add them to autohost or smth so they would work.
как добавить новый сайт что бы запрет и на него работал. это для анимего и джутсу их заблокали и только впн поможет с ними. вопрос в том как добавить их в хост лист или типо того что бы они заработали

**Comment by electrifying** [comment]
Ну просто хостлист открываешь и добавляешь url

**Comment by numbereleven-a** [comment]
> how to add new site so zapret would work on it too. it for animego and jut.su. they been blocked and only vpn could help with that. the question is how i add them to autohost or smth so they would work. как добавить новый сайт что бы запрет и на него работал. это для анимего и джутсу их заблокали и только впн поможет с ними. вопрос в том как добавить их в хост лист или типо того что бы они заработали

Добавить сайты которые должны обходиться в zapret-hosts-user.txt, поддомены учитываются автоматически, в стратегиях указать параметр `<HOSTLIST>`.
Если нужно чтобы какие то сайты не попадали под случайный обход то в zapret-hosts-user-exclude.txt.
Способ работает для режимов фильтрации MODE_FILTER=hostlist или MODE_FILTER=autohostlist.
Следует учесть нулевую фазу, если в стретегиях используется дурение: `synack, syndata, wsize, wssize` то на эту фазу не действуют фильтры по hostlist.

**Reply by kiiroless** [reply]
так я дурачек и не сильно шарю как работает запрет. файл который ты указал у меня не нашелся, и скорее всего потому что я установил запрет по быстрому старту для винды. по поводу второй части спасибо большое но опять же у меня отсутствуют данные файлы zapret-hosts-user.txt и zapret-hosts-user-exclude.txt.  если как то можно это исправить буду благодарен

**Reply by numbereleven-a** [reply]
> так я дурачек и не сильно шарю как работает запрет. файл который ты указал у меня не нашелся, и скорее всего потому что я установил запрет по быстрому старту для винды. по поводу второй части спасибо большое но опять же у меня отсутствуют данные файлы zapret-hosts-user.txt и zapret-hosts-user-exclude.txt. если как то можно это исправить буду благодарен

виндоус приложение не использую, не знаю как там, но думаю похоже

**Reply by kiiroless** [reply]
![image](https://github.com/user-attachments/assets/c1c80038-bd4e-4a5d-8039-92f80feb51b7)
![image](https://github.com/user-attachments/assets/e6cf98ee-b366-4989-a7f5-f9be62abb217)

так вроде пикча загрузилась выглядит папка сама вот так.
чел написал до тебя что в хостлист нужно добавить но даже так оно не помогает. идеи?

**Comment by kaskaderskiri** [comment]


**Reply by kaskaderskiri** [reply]
https://ntc.party/t/%D0%B1%D0%BB%D0%BE%D0%BA%D0%B8%D1%80%D0%BE%D0%B2%D0%BA%D0%B0-telegram-%D0%B8-whatsapp-%D0%B7%D0%B2%D0%BE%D0%BD%D0%BA%D0%BE%D0%B2/18333
тут смотрел
